packages/ai/core/tool/mcp.ts

```diff
@@ -0,0 +1,209 @@
+import { Client } from '@modelcontextprotocol/sdk/client/index.js';
```

Comment:
strange that the imports are with `.js` - is this intentional?

Reply:
yeah that was the auto import!